includes/utils/class-amp-dom-utils.php

```diff
@@ -285,6 +333,14 @@ public static function get_content_from_dom_node( $dom, $node ) {
 
 		$html = self::restore_amp_bind_attributes( $html );
 
+		// Restore amp-mustache placeholders which were replaced to prevent URL-encoded corruption by saveHTML.
```

Comment:
Perhaps something like the prototype replacement below would have been sufficient:

``` PHP
$html = preg_replace_callback(
	"#href=[\"|'](%7B.*%7D)?[\"|']+#",
	function( $matches ) {
		return urldecode( $matches[0] );
	},
	$html
);
```

Reply:
I didn't want to do replacements on encodings that were original. And I wanted to limit the replacements to braces.